React/Modules/RCTDeviceInfo.m

```diff
@@ -53,15 +47,24 @@ - (void)setBridge:(RCTBridge *)bridge
 
   // Don't use RCTScreenSize since it the interface orientation doesn't apply to it
   CGRect screenSize = [[UIScreen mainScreen] bounds];
-  NSDictionary *dims = @{
+  NSDictionary *screenDims = @{
                          @"width": @(screenSize.size.width),
                          @"height": @(screenSize.size.height),
                          @"scale": @(RCTScreenScale()),
                          @"fontScale": @(bridge.accessibilityManager.multiplier)
                          };
+
+  CGRect window = [[UIApplication sharedApplication] keyWindow].rootViewController.view.bounds;
```

Comment:
Can we just use `[[UIApplication sharedApplication] keyWindow].bounds`?

Why do we need to subscribe for `RCTUIManagerDidUpdateAvailableSizeNotification`? Is your app hybrid (uses not only RN but also native views and view controllers)?

I hope, should be the better/easier way to subscribe for UIWindow size changing notification.